wpeverest/everest-forms
# EVF-202 Fix - Color Picker in Icon Color of Ratings field

### All Submissions:






### Changes proposed in this Pull Request:


Previously Color Picker is not showing in Icon color of Rating Field. This PR fixes this issue.

### How to test the changes in this Pull Request:

1. In Admin Side create a form with the rating field and it's field setting go to advance options.
2. Check the color picker is showing or not in Icon Color section.
3. Verify Color picker is working properly or not.

### Types of changes:




### Other information:




### Changelog entry

>Fix - Color Picker in Icon Color of Ratings field
